src/main/java/com/epam/controller/HotelController.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.controller;
+
+import com.epam.model.Hotel;
+import com.epam.service.HotelService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Controller
+public class HotelController {
+    private final HotelService hotelService;
+
+    public HotelController(HotelService hotelService) {
+        this.hotelService = hotelService;
+    }
+
+    @GetMapping("/hotels")
+    public String hotelPage(ModelMap modelMap) {
+        return "hotels";
+    }
+
+    @PostMapping("/hotels")
+    public ModelAndView addHotel(@RequestParam(name = "name") String name,
+                                 @RequestParam(name = "country") String country,
+                                 @RequestParam(name = "city") String city,
+                                 @RequestParam(name = "stars") Integer stars,
+                                 ModelAndView modelAndView,
+                                 RedirectAttributes redirectAttributes) {
+        redirectAttributes.addFlashAttribute("add_status", "successful");
+        Pattern pattern = Pattern.compile("^[a-zA-Z ]*$");
+        Matcher countryMatcher = pattern.matcher(country);
+        Matcher cityMatcher = pattern.matcher(city);
+        if (name.equals("")) {
```

Comment:
seems like switch

Reply:
I think it is more like there were some extra elses. Please look again, I've changed it

Reply:
okay, but please, replace it logic to the service layer and add tests for it

Reply:
done